pkg/miekg/miekg.go

```diff
@@ -74,6 +77,16 @@ type GlobalLookupFactory struct {
 	BlMu           sync.Mutex
 }
 
+// Lookup client interface for helping in mocking
+type LookupClient interface {
+	ProtocolLookup(s *Lookup, q Question, nameServer string) (interface{}, zdns.Trace, zdns.Status, error)
+}
+
+// Check whether the status is safe
+func SafeStatus(status zdns.Status) bool {
```

Comment:
If this is used anywhere else (I'm not sure if the modules will have any use for this, or the CLI) it may be better defined in the util package.

Reply:
Moved to util